MongoTrait/Metaable.php

```diff
@@ -36,35 +47,85 @@
     protected $metaFollow = false;
 
     /**
-     * @param string $metaKeywords
+     * @param Collection $metaKeywords
      */
-    public function setMetaKeywords($metaKeywords)
-    {
+    public function setMetaKeywords(Collection $metaKeywords) {
         $this->metaKeywords = $metaKeywords;
     }
 
     /**
+     * @param Collection $metaDescription
+     */
+    public function setMetaDescriptions(Collection $metaDescription) {
+        $this->metaDescriptions = $metaDescription;
```

Comment:
@djamnazi : You need to check the pre-conditions here: the collection could contains anything, you need to check that each element of the collection respects what you're expecting for a metaDescription